### ISSUE TITLE
Highlight color does not meet minimum contrast threshold

### ISSUE THREAD

**Issue by VirginiaBalseiro** [dokieli/dokieli#506]
The light yellow (#fdffdd) does not have enough contrast against the
background (this should be at least 3:1).

Example from the docs view (info button) for Share: 

<img width="818" height="995" alt="Image" src="https://github.com/user-attachments/assets/514c1df6-49e2-4613-893d-186d88f5a168" />

See: https://www.w3.org/WAI/WCAG22/Understanding/non-text-contrast

**Comment by VirginiaBalseiro** [comment]
SC 1.4.11 Non-Text Contrast applies to non-text visual cues (such as things like borders, icons, focus indicators, and UI states). It does not normally apply to text highlighting, because highlighting is part of the text's presentation, so regular text contrast rules (like 1.4.3 Contrast Minimum) apply instead, unless the highlight is functioning as a UI component (like a tag or badge, which in this case it is not).

See: 

> The intent of this success criterion is to ensure that user interface components (i.e., controls) and meaningful graphics are distinguishable by people with moderately low vision. The requirements and rationale are similar to those for large text in [1.4.3 Contrast (Minimum)](https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum). Note that this requirement does not apply to inactive user interface components.
